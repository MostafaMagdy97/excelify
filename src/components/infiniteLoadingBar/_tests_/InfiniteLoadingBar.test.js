import React from 'react';
import { configure, render } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import 'jest-enzyme';
import 'jest-styled-components';

import InfiniteLoadingBar from '../InfiniteLoadingBar';

configure({ adapter: new Adapter() });

describe('InfiniteLoadingBar', () => {
  it('should render', () => {
    const component = <InfiniteLoadingBar />;
    const tree = render(component);
    expect(tree).toMatchSnapshot();
  });
});
