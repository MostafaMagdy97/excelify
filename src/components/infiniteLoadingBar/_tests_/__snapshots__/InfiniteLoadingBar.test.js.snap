// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfiniteLoadingBar should render 1`] = `
.c0 {
  background-clip: padding-box;
  background-color: #f5f5f5;
  border-radius: 15px;
  display: block;
  height: 4px;
  overflow: hidden;
  position: relative;
}

.c1 {
  background-color: #06BEE1;
  border-radius: 15px;
}

.c1::before {
  -webkit-animation: wwaAf 1.75s cubic-bezier(0.65,0.815,0.735,0.395) infinite;
  animation: wwaAf 1.75s cubic-bezier(0.65,0.815,0.735,0.395) infinite;
  background-color: inherit;
  bottom: 0;
  content: '';
  left: 0;
  position: absolute;
  top: 0;
  will-change: left,right;
}

.c1::after {
  -webkit-animation: kBRfEP 1.75s cubic-bezier(0.165,0.84,0.44,1) infinite;
  animation: kBRfEP 1.75s cubic-bezier(0.165,0.84,0.44,1) infinite;
  -webkit-animation-delay: 1.15s;
  animation-delay: 1.15s;
  background-color: inherit;
  bottom: 0;
  content: '';
  left: 0;
  position: absolute;
  top: 0;
  will-change: left,right;
}

<div
  class="c0"
>
  <div
    class="c1"
  >
    
  </div>
</div>
`;
